service/src/main/java/com/ge/predix/acs/privilege/management/SubjectPrivilegeManagementController.java

```diff
@@ -124,7 +119,7 @@ private void failIfParentsSpecified(final List<BaseSubject> subjects) {
         }
 
         if (subject == null) {
-            return notFound();
```

Comment:
Same comment as for resource management.  Not sure if returning an error is the best approach here. May be returning an empty body alone with Not Found status is better?